src/types/job.rs

```diff
@@ -29,7 +29,7 @@ pub struct JobDescriptor {
     pub label: String,
     pub num_dependencies: u32,
     pub score: f64,
-    pub packages: Vec<PackageDescriptor>,
+    pub packages: PackageDescriptorsOrPurls,
```

Comment:
This is a type returned by API in the [get user jobs](https://api.staging.phylum.io/api/v0/swagger/index.html#/Jobs/get_user_jobs) endpoint. How will it decide whether to send PURLs or descriptor objects here?

If it is going to start return PURLs, this is a breaking API change. Maybe this should be accompanied by a version number bump (i.e., `/api/v0` returns descriptor objects while `/api/v1` returns PURLs)?

Reply:
It will send descriptors until they are being deprecated. Unfortunately, because of the way this repository is used we need to add impossible cases to it like this and then make them possible later.